code/Task_List/app/src/main/java/aleksandr_logunov/task_list/MainActivity.kt

```diff
@@ -20,7 +20,7 @@ class MainActivity : BaseActivity<ActivityMainBinding>() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         viewBinding = ActivityMainBinding.inflate(layoutInflater)
-        with (viewBinding) {
+        with (viewBinding) { // better have this in a base class
```

Comment:
I don't get how to move viewBinding related code into base class because it's heavily dependent on views so it won't be the same for different activities